Sources/docc/DocCDocumentation.docc/adding-code-snippets-to-your-content.md

```diff
@@ -0,0 +1,213 @@
+# Adding Code Snippets to your Content
+
+@Metadata {
+    @Available("Swift", introduced: "5.7")
```

Comment:
This is different from the `introducedVersion` on the Snippet type. We should update one of these values so that they are both correct.

Reply:
I updated it to 5.7 - while it was technically released with 5.6, it wasn't really operational until after 5.7 from what I saw due to a couple lingering issues. I can flip it back to 5.6 if you'd prefer.